### PR TITLE
{bio}[foss/2019a,intel/2019a] RELION v3.0.7

### PR DESCRIPTION
**Pull request by henkela**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
I'm working on a slight variation of this with more dependencies, and also the CUDA version.
Should be done in a few days depending on interrupt level...

File: easybuild/easyconfigs/r/RELION/RELION-3.0.7-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+builddependencies = [('CMake', '3.13.3')]
+
+dependencies = [
+    ('X11', '20190311'), 
+    ('FLTK', '1.3.5'),
+    ('LibTIFF', '4.0.10'),
+    ('tbb', '2019_U4', '-GCCcore-8.2.0'),
+]
+
+configopts = "-DCMAKE_SHARED_LINKER_FLAGS='-lpthread'  -DMPI_INCLUDE_PATH=$EBROOTOPENMPI/include "
```
Comment (timeu) [inline easybuild/easyconfigs/r/RELION/RELION-3.0.7-foss-2019a.eb]: For 2019.02 intel toolchain I had to use following config opts:
` -DMPI_INCLUDE_PATH=$EBROOTIMPI/intel64/include` and `"-DMPI_C_COMPILER=$EBROOTIMPI/intel64/bin/mpiicc -DMPI_CXX_COMPILER=$EBROOTIMPI/intel64/bin/mpiicpc "`

**Comment by smoors** [comment]
@akesandgren what's the status of your variation?
if you don't have time, maybe post what you have currently so @henkela can finish it?

**Comment by akesandgren** [comment]
Ah yes.... coming up...

**Comment by akesandgren** [comment]
See #9096

File: easybuild/easyconfigs/r/RELION/RELION-3.0.7-intel-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+dependencies = [
+    ('X11', '20171023'),
+    ('FLTK', '1.3.4'),
+    ('LibTIFF', '4.0.9'),
+    ('tbb', '2019_U4'),
+]
+
+configopts = "-DCMAKE_SHARED_LINKER_FLAGS='-lpthread'  -DMPI_INCLUDE_PATH=$EBROOTIMPI/intel64/include "
+configopts += "-DMPI_C_COMPILER=$EBROOTIMPI/intel64/bin/mpiicc -DMPI_CXX_COMPILER=$EBROOTIMPI/intel64/bin/mpiicpc "
```
Comment (akesandgren) [inline easybuild/easyconfigs/r/RELION/RELION-3.0.7-intel-2019a.eb]: Didn't think of that before, but the way to do this is to specify the 'usempi': True toolchain option.
Then you don't need to set those MPI_xxx variables.

**Comment by boegel** [comment]
@henkela This is currently one of the oldest open PRs, and it hasn't moved much recently...

Do you think it's still worth the effort to try and get this merged? If not, please close the PR.

**Comment by akesandgren** [comment]
And we have https://github.com/easybuilders/easybuild-easyconfigs/pull/16527 for a newer version

**Pull request closed without merging**